pkg/commands/backend/create.go

```diff
@@ -203,6 +214,34 @@ func (c *CreateCommand) Exec(_ io.Reader, out io.Writer) error {
 	if c.sslSNIHostname.WasSet {
 		input.SSLSNIHostname = &c.sslSNIHostname.Value
 	}
+	if c.tcpKaEnable.WasSet {
+		var tcpKaEnable bool
+
+		switch c.tcpKaEnable.Value {
+		case "true":
+			tcpKaEnable = true
+		case "false":
+			tcpKaEnable = false
+		default:
+			err := errors.New("'tcp-ka-enable' flag must be one of the following [true, false]")
+			c.Globals.ErrLog.Add(err)
+			return err
```

Comment:
The unit test should have a scenario for this, to ensure that invalid values are rejected; it's OK to add three scenarios to cover all three possibilities.

Reply:
Done in 208e61da518bc6cd44cf0a7563eb0449b1aff2fb!